docs/release-notes/Percona-Everest-1.2.0-(2024-09-09).md

```diff
@@ -0,0 +1,138 @@
+
+- [RBAC resources and privileges](https://docs.percona.com/everest/administer/rbac.html#rbac-resources-and-privileges): Resources are the entities or objects within Percona Everest that require controlled access. Privileges specify the particular actions that a role is able to perform on a resource.
+
+- [Policy definition](https://docs.percona.com/everest/administer/rbac.html#policy-definition-in-rbac): RBAC policies are the rules and guidelines that define how roles, permissions, and users are managed within RBAC.
+
+
+    The policy definition in Percona Everest is:
+
+    ```sh
+    p, <subject>, <resource-type>, <action>, <resource-name>
+    ```
+
+- [Role assignment](https://docs.percona.com/everest/administer/rbac.html#assigning-roles-to-users): Assigning specific roles to individual users within Percona Everest is crucial for the roles to be effective.
+
+    The syntax for assigning a role is as follows:
+
+    ```sh
+    g, username, rolename
+    ```
+
+Explore our comprehensive [documentation](https://docs.percona.com/everest/administer/rbac.html) for everything you need to know about RBAC.
+
+
+### Percona Everest 1.2.0: A deep dive into Breaking API changes
+
+Beginning with Percona Everest v1.2.0, breaking changes are being introduced to the API for `monitoring-instances` and `backup-storages` resources. These updates include:
+
+- Before the launch of Percona Everest 1.2.0, the resources `monitoring-instances` and `backup-storages` had a global scope. Percona Everest used a `.spec.allowedNamespaces` field to control access to these global resources. This field defined the namespaces where the resources could be accessed, thus providing some degree of access control.
+
+- With the upgrade to Percona Everest version 1.2.0, the transition from global scope to the designated namespaces for these resources is an important change in the way access control is managed. This improves security as the resources are only accessible within their designated namespaces. The database clusters can only use `monitoring-instances` and `backup-storages` located within the same namespace as the cluster.
+
+- When upgrading to 1.2.0 using the CLI command `everestctl upgrade`, all your existing `backup-storages` and `monitoring-instances` will be automatically migrated to the namespaces specified in their `.spec.allowedNamespaces` fields.
+
+    !!! note
+        After the upgrade to Percona Everest 1.2.0, you will only be able to access these resources through the new API endpoints.
+
+    Check out our [documentation](docs.percona.com/everest//administer/api_rbac.html#changes-in-the-percona-everest-apis) for in-depth details on the Breaking API changes.
+
+
+
+
+### Improved multiple operator upgrades
+
+
+Starting with Percona Everest 1.2.0, we have the capability to upgrade all the database operators concurrently with their components across any namespace using our intuitive UI with just a single click.
```

Comment:
I think this is slightly misleading in the sense that this option is not just a new capability, but the only option available. Earlier we would allow (or at least make it seem like) upgrading just a single operator, but that is no longer an option now because of a technical limitation

Reply:
ok. Lets discuss with @recharte.

Reply:
@mayankshah1607 updated pls check